Decentralized-Autonomous-Organizations.md

```diff
@@ -875,6 +890,9 @@ export default function Home() {
         getDAOTreasuryBalance();
         getUserNFTBalance();
         getNumProposalsInDAO();
+        if (parseFloat(treasuryBalance) > 0) {
```

Comment:
This should not be here. Why would you withdraw ether everytime balance is > 0. This should be on a button click or something

Reply:
Yes, you are right. Let me make some change like if Owner and contract has some eth then we render the withdraw button, else we don't render it. That would be fine ?

Reply:
Hi @haardikk21 , 

I have made the changes and committed them. Can you please review and test them from your side ?

Please let me know what do you think.

Some Screen Shots for reference:

View for Owner :
-----------------
<img width="1237" alt="Screenshot 2022-09-30 at 2 00 30 PM" src="https://user-images.githubusercontent.com/55076843/193237587-cf40ebad-1da3-4525-b0aa-212d8f3ae83e.png">

View for owner after withdrawn:
-------------------------------

<img width="1191" alt="Screenshot 2022-09-30 at 2 02 56 PM" src="https://user-images.githubusercontent.com/55076843/193237832-64ad75dc-5c81-4a9e-9a6c-466423ca1c08.png">


View for no-owner: 
-------------------
<img width="1073" alt="Screenshot 2022-09-30 at 1 57 09 PM" src="https://user-images.githubusercontent.com/55076843/193237988-4a61a7ea-ab23-4ba4-9b6b-881ff855f860.png">